doc/themes/scikit-learn/layout.html

```diff
@@ -272,9 +272,9 @@
     {% endif %}
 
     {%- if pagename != "install" %}
-      <p class="doc-version">This documentation is for {{project}} <strong>version {{ release|e }}</strong> &mdash; <a href="http://scikit-learn.org/stable/support.html#documentation-resources">Other versions</a></p>
+      <p class="doc-version">This documentation is for {{project}} <b style="font-size: 110%;">version {{ release|e }}</b> &mdash; <a href="http://scikit-learn.org/stable/support.html#documentation-resources">Other versions</a></p>
     {%- endif %}
-    <p class="citing">If you use the software, please consider <a href="{{pathto('about')}}#citing-scikit-learn">citing scikit-learn</a>.</p>
+    <p class="citing"><b><a href="about.html#citing-scikit-learn" style="font-size: 110%;">Cite us </a></b>if you use the software.</p>
```

Comment:
seeing as this takes up 2 lines anyway (thus perhaps no harm in adding some more text), how's "Please cite us if you use scikit-learn"?